src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx

```diff
@@ -894,7 +894,7 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
             isLocalFile && Str.isPDF(receiptFilename) ? (
                 <PDFThumbnail
                     // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
-                    previewSourceURL={resolvedThumbnail as string}
```

Comment:
Why does this fix this? I would imagine that the thumbnail should be the source for <PDFThumbnail?

Reply:
Could we see why this is failing? or what is even in resolvedThumbnail?

Perhaps we just change it to this
resolvedThumbnail || resolvedReceiptImage || ''

Reply:
> Perhaps we just change it to this
> resolvedThumbnail || resolvedReceiptImage || ''

No need IMO, local files don't have thumbnail:

https://github.com/Expensify/App/blob/3cf796caf8c11b2cb8e4b68d0d1fea3ad2550df4/src/libs/ReceiptUtils.ts#L47-L49